src/Component/RequestBuilder/EdisMax.php

```diff
@@ -46,6 +46,7 @@ public function buildComponent($component, $request)
         $request->addParam('bf', $component->getBoostFunctions());
         $request->addParam('boost', $component->getBoostFunctionsMult());
         $request->addParam('uf', $component->getUserFields());
+        $request->addParam('sow', null === ($sow = $component->getSplitOnWhitespace()) ? null : ($sow ? 'true' : 'false'));
```

Comment:
The special handling of null and the string representation of booleans is not required here. We already do that internally in addParam().